cpp/include/kvikio/shim/libcurl.hpp

```diff
@@ -249,12 +250,31 @@ class CurlHandle {
   }
 };
 
+namespace detail {
+/**
+ * @brief Fix Conda's manipulation of __FILE__.
+ *
+ * Conda manipulate the path information in its shared libraries with the results that the C macro
```

Comment:
```suggestion
 * Conda manipulates the path information in its shared libraries with the results that the C macro
```

Add link to the relevant conda docs? https://docs.conda.io/projects/conda-build/en/latest/resources/make-relocatable.html